course_content/notebooks/iris_intro.ipynb

```diff
@@ -436,31 +403,36 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Sometimes it is desirable to add bounds to a coordinate that doesn't have any. This is often the case for creating \"block\" type plots where a coordinate should be able to represent an interval of values, rather than a single point. The ``guess_bounds`` method on a coordinate is useful in this regard. For example, the latitude coordinate previously obtained does not have bounds, but we can either set some manually, or use the ``guess_bounds`` method:"
+    "### Discussion: the cube object\n",
+    "\n",
+    "* What advantages over NumPy arrays do Iris cubes provide?\n",
+    "* What information does an Iris cube contain?\n",
+    "* How do Iris cubes and coordinates relate to each other?"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
+   "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "print lat.points[:4]\n",
-    "print lat.bounds\n",
-    "if lat.bounds is None:\n",
-    "    lat.guess_bounds()\n",
-    "print lat.bounds[:4]"
+    "## Loading and saving data\n",
+    "\n",
+    "**Learning outcome**: by the end of this section, you will be able to use Iris to load datasets from disk as Iris cubes and save Iris cubes back to disk."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Exercise 1\n",
+    "### The three load functions: load, load_cube and load_cubes\n",
+    "\n",
+    "There are three main load functions in Iris: ``load``, ``load_cube`` and ``load_cubes``.\n",
```

Comment:
I feel like this is a bit repetitive.  Maybe don't list the functions in the section title?

Reply:
> this is a bit repetitive

Good point. I like the suggestion of removing the function specifics from the heading 👍